public/app/ui/pages/about/AboutPage.js

```diff
@@ -3,9 +3,14 @@
 
     define(
         [
+            'util/container'
         ],
-        function () {
-            return function (context) {
+        function (container) {
+            return function () {
+                this.ready = function (callback) {
```

Comment:
When the view is ready, hide the loading overlay.  This only really affects pages being loaded directly by URL, i.e. the first page loaded by the app, because the loading overlay is initially visible (i.e. before javascript kicks in).  If navigating from another page to the About page (or Home or Error page), the loading overlay is never even displayed, since there is nothing to load (the markup is baked into the app).